sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpFilter.cs

```diff
@@ -57,14 +57,14 @@ public static string BuildFilterExpression(EventPosition eventPosition)
         {
             // Build the filter expression, in the order of significance.
 
-            if (!string.IsNullOrEmpty(eventPosition.Offset))
+            if (!string.IsNullOrEmpty(eventPosition.SequenceNumber))
```

Comment:
This, I'd revert.  We should not see any instances with multiple properties set under normal circumstances.   The processor uses the `FromSequenceNumber` and `FromOffset` factories, so the ordering here won't impact it.

Reply:
Updated!